src/utils/pgwire/src/pg_extended.rs

```diff
@@ -133,53 +133,69 @@ impl PgPortal {
         self.row_description.clone()
     }
 
-    pub async fn execute<SM: SessionManager>(
+    pub async fn execute<SM: SessionManager, S: AsyncWrite + AsyncRead + Unpin>(
```

Comment:
Why add processing execute results in this function?

BTW seems like some comments are also deleted

Reply:
Because it's different to process execute results from simple query mode and extended query mode. For extended query mode, we need to return the partition of the result and cache reset when row_limit is not 0. I think it's more clear to just process the result separately. 
Actually, I rewrite the process way of the extended query mode. In the past, we need to cache the result, but now we can just consume the row we need and reuse the stream.